physics/src/modules/ThermoIce0.cpp

```diff
@@ -150,9 +148,10 @@ void ThermoIce0::calculateElement(size_t i, const TimestepTime& tst)
         qio[i] += hice[i] * bulkLHFusionIce / tst.step + hsnow[i] * bulkLHFusionSnow / tst.step;
 
         // No ice, no snow and the surface temperature is the melting point of ice
+        cice[i] = 0.;
         hice[i] = 0.;
         hsnow[i] = 0.;
-        tice.zIndexAndLayer(i, 0) = Ice::Tm;
+        tice.zIndexAndLayer(i, 0) = 0.;
```

Comment:
I don't understand the significance of cice[i] = 0 and changing Ice::Tm ---> 0. To me L154 looks like an error given the comment on L150. Please satisfy yourself that this is correct / update or expand on the comment. 

Would also help if 'c' and 'h' and 't' were converted to useful prefixes and/or we used camel case as is required by the webkit style / nextsim-dg coding conventions

Reply:
Actually, this makes sense. I have changed it the value of Ice::Tm in Celsius. Which is still zero, but is now more explicit in the code.